projboard/tests/test_user.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from projboard.models import User
+
+
+NAME = "RAWAD"
+NICKNAME = "R1998"
+EMAIL = "rawad@gmail.com"
+PASSWORD = "rawad!@#"
+
+
+@pytest.fixture
+def generate_user():
+    user1 = User.create_user(name="RAWAD", nickname="R1998", email="rawad@gmail.com", password="rawad!@#")
+    user2 = User.create_user(name="John", nickname="J99", email="John@gmail.com", password="John123")
```

Comment:
Why do we need the second user? it is not been used in the tests.

Reply:
+1
In all test below, you are using only one generate_user[0]

I think you should also search for the user in db with the key field, or model method, isnt it? 

the search can be taken from fixture data

Reply:
Done